dot/sync/chain_processor.go

```diff
@@ -92,8 +92,8 @@ func (s *chainProcessor) processReadyBlocks() {
 
 		if err := s.processBlockData(*bd); err != nil {
 			// depending on the error, we might want to save this block for later
-			if !errors.Is(err, errFailedToGetParent) {
-				logger.Errorf("block data processing for block with hash %s failed: %s", bd.Hash, err)
+			if !errors.Is(err, errFailedToGetParent) && !errors.Is(err, blocktree.ErrParentNotFound) {
+				logger.Tracef("block data processing for block with hash %s failed: %s", bd.Hash, err)
```

Comment:
Change this back to `Errorf` since we should see less of them with the additional condition you added right?

```suggestion
				logger.Errorf("block data processing for block with hash %s failed: %s", bd.Hash, err)
```

Reply:
We have a whole lot of them. Look at kusama logs on staging.  It is polluting logs.

Reply:
I am going to revert this change and merge this. If I still see this error being spammed all over logs, I will just add it to one of the future PRs. It takes about a days to get till the logs that show such errors (if I remember right)